csrc/transformer/inference/csrc/pt_binding_unittest.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The Microsoft DeepSpeed Team
+*/
+
+#include <c10/cuda/CUDAStream.h>
+#include <torch/extension.h>
+#include <stdexcept>
```

Comment:
Some of these includes aren't necessary, let's pull all that aren't necessary to compile.

Reply:
removed unnecessary includes